lnst/Tests/TrafficControl.py

```diff
@@ -31,18 +35,44 @@ def run(self) -> bool:
 
     async def run_instances(self) -> list[dict]:
         tc_exec = shutil.which("tc")
+        cpu_bind = self.params.get("cpu_bind")
+        cpu_bind_gen: Iterator[Optional[list[int]]]
+        if cpu_bind is not None:
```

Comment:
`is not None` is imo not needed. If the list is empty, the result would be empty generator, or am I wrong?

Reply:
You're right, fixed and also moved to a separate method.